content/blog/2020/08/2020-08-04-cdf-graduation.adoc

```diff
@@ -0,0 +1,167 @@
+This status is official effective Aug 03, 2020.
+
+In this article, we will discuss what the CDF membership and graduation mean to the Jenkins community and what are the future steps for the project.
+If you are interested to know more,
+see the link:TODO[announcement on the CDF website]!
+
+== How does CDF membership help us?
+
+About 18 months ago, Jenkins link:/blog/2019/03/12/cdf-launch/[became one of the CDF founding projects],along with Jenkins X, Spinnaker and Tekton.
+A new foundation was formed to provide a vendor-neutral home for open source projects used for Continuous Delivery and Continuous Integration.
+Special interest groups were started to foster collaboration between projects and end user companies,
+most notably 
+link:https://github.com/cdfoundation/sig-interoperability[Interoperability],
+link:https://github.com/cdfoundation/sig-mlops[MLOps] and
+link:https://github.com/cdfoundation/sig-security[Security] SIGs.
+Also, a link:https://cd.foundation/ambassador-program-overview-application/[Community Ambassador] role was created to organize local meetups and to provide public-facing community representatives.
+Many former Jenkins Ambassadors and other contributors are now CDF Ambassadors, and they promote Jenkins and other CDF projects there.
+
+Thanks to this membership we addressed key link:/projects/infrastructure/[project infrastructure] needs.
+Starting from Jan 2020, CDF covers a significant part of the infrastructure costs
+including our services and CI/CD instances running on Microsoft Azure.
+CDF provided us with legal assistance required to get code signing keys for the Jenkins project.
+Thanks to that, we were able to switch to a new link:https://github.com/jenkins-infra/release[Jenkins Release Infrastructure].
+The foundation sponsors the Zoom account we use for link:/events/online-meetup/[Jenkins Online Meetups] and community meetings.
+In the future we consider reducing maintenance overheads by switching some of our self-hosted services to equivalents provided by the Linux Foundation to CDF members.
+
+Another important CDF membership benefit is community outreach and marketing. 
+CDF helped us to establish connections with other CI/CD projects and end user companies.
+Through CDF we have access to the link:https://jenkins.devstats.cd.foundation/[DevStats service].  It provides community contribution statistics and helps us track trends and discover areas for improvement.
+On the marketing side, the foundation organizes webinars, podcasts and newsletters.
+Jenkins is regularly represented there.
+CDF also runs the meetup.com professional account which is used by local Jenkins communities for
+link:/projects/jam/[CI/CD and Jenkins Area Meetups].
+Last but not least, the Jenkins community is also represented at virtual conferences where CDF has a booth.
+All of that helps to grow Jenkins visibility and to highlight new features and initiatives in the project.
+
+== Why did we graduate?
```

Comment:
```suggestion
== Why do we graduate?
```

Not sure, would appreciate your feedback @MarkEWaite

Reply:
> Not sure, would appreciate your feedback @MarkEWaite

```
+ == Why are we graduating?
```

Idea is to keep the reader in the present tense for the question, then answer the question in the paragraph.  If not present tense, then my personal preference is "Why did we graduate?"